runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/RuntimeOutput.kt

```diff
@@ -0,0 +1,13 @@
+package io.openruntimes.kotlin;
+
+class RuntimeOutput(body: String, statusCode: Int, headers: MutableMap<String, String>) {
+    var body: String
+    var statusCode: Int
+    var headers: MutableMap<String, String>
+
+    init {
+        this.body = body
+        this.statusCode = statusCode
+        this.headers = headers
+    }
+}
```

Comment:
This can be replaced with a data class:

```suggestion
data class RuntimeOutput(val body: String, val statusCode: Int, val headers: MutableMap<String, String>)
```